### PR TITLE
3/identify unpublished pages in tree

### PR DESCRIPTION
**Pull request by stuartromanek**
https://apostrophecms.atlassian.net/browse/A30U-920?atlOrigin=eyJpIjoiZjZlNmQ2YWRmZjIyNDdkZTk1N2MwNTkwMjNmOTY5MDciLCJwIjoiaiJ9

First part of this ^ .. second part should be its own ticket as it would (probably) impact the pages `getAll` query more significantly than adding an additional projection property.

### CODE REVIEW

File: modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue [hunk 1]
```diff
@@ -84,6 +84,14 @@ export default {
             label: 'Page Title',
             name: 'title'
           },
+          {
+            label: 'Published',
+            name: 'lastPublishedAt',
+            value: {
+              true: '',
+              false: 'Unpublished'
```
Comment (abea) [inline modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: Maybe this should be `indicator`? `value` is more of a computer-read property. I realize `label` is taken here.

**Review by abea — COMMENTED** [review]
It is clever. Given that this is the visible label for the value, `value` is striking me as not quite right, but there's not a perfect alternative that comes right to mind.

File: modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue (hunk 1 above)
Comment (stuartromanek) [reply modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: yeah value didn't seem right to me .. what about `label` being able to be both an object and a string and figuring it out down there at the template?

Comment (abea) [reply modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: This is describing the column. `columnHeader` could be appropriate, [based on the related aria-role name](https://w3c.github.io/aria/#columnheader). Then `label` is freed up. Maybe it'd be good to specify both better: `columnHeader` or `header` and then `cellLabel`.

Comment (stuartromanek) [reply modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: oo that would not really be that same thing ... maybe `label` should change to `heading` to be more in line with how a table gets constructed?

then i could take `label` for this

Comment (abea) [reply modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: Simply making `label` do more work doesn't seem right.

Comment (stuartromanek) [reply modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: fine w the explicit versions of `columnHeader` and `cellLabel`

Comment (boutell) [reply modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue:92]: I like the improved property names.

File: modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { detectDocChange } from 'Modules/@apostrophecms/schema/lib/detectChange'
 // Vue 3 using the composition API. - AB
 
 import klona from 'klona';
+import _ from 'lodash';
```
Comment (boutell) [inline modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js:8]: Use `Array.from(new Set(array))` to unique-ify an array without hauling in lodash. isObject can be `(typeof foo === 'object')`, isString can be `(typeof foo === 'string')`.

**Review by boutell — CHANGES_REQUESTED** [review]

**Review by boutell — APPROVED** [review]

**Pull request merged**